### ISSUE TITLE
Xi Chen

### ISSUE THREAD

**Issue by bensoltoff** [uchicago-computation-workshop/2019_spring_conf#22]

**Comment by bensoltoff** [comment]
Lots of models presented here. Maybe simplify it for such a short presentation. It's hard to follow along what all of them do

**Comment by rickecon** [comment]
* Good description of the computational methods that you add to your analysis.
* Good behavioral testing. I like that you went after "why did this model perform better?"
* The two lines in each of your plots are too close to each other to give much information. You might consider getting rid of those.
